index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link rel="stylesheet" href="./styles.css">
+    <title>Portfolio | Debabrata Mandal</title>
+</head>
+
+<body>
+    <section>
+        <header>
+            <div id="logo">Mr. Mandal</div>
+            <img id="menu-icon" alt="menu icon" src="./images/menu_icon.svg">
+        </header>
```

Comment:
- [ ] [Optional] the toolbar section should be separated from the headline section. Kindly take it out of the section tag as this will improve your page SEO